src/base/src/db3_document.rs

```diff
@@ -0,0 +1,197 @@
+use bson::spec::BinarySubtype;
+use bson::Document;
+use bson::{Binary, Bson, RawDocumentBuf};
+use db3_error::{DB3Error, Result};
+use fastcrypto::traits::ToFromBytes;
+use serde_json::{Map, Value};
+
+#[derive(Debug)]
+pub struct DB3Document {
+    doc: Document,
+}
+impl DB3Document {
+    pub fn into_bytes(&self) -> Vec<u8> {
+        let row_doc = RawDocumentBuf::from_document(&self.doc).unwrap();
+        row_doc.into_bytes()
+    }
+    pub fn add_documentId(&mut self, docId: &str) {
+        self.doc.insert("_docId", docId);
```

Comment:
add a document_constants.rs to define the docId , txId and ownerAddr

Reply:
sure. I will add in the next revision